### ISSUE TITLE
lodash.cloneDeep

### ISSUE THREAD

**Issue by IMeyers2020** [lodash/lodash#5624]
Hey there,

I am currently working on a project that is using lodash. It is great, but I don't really need the whole package as it is pretty bulky. After looking into it, I found this page explaining how to use a modularized version of the individual packages: https://stackoverflow.com/questions/43479464/how-to-import-a-single-lodash-function

And then the specific npm package would be found here: https://www.npmjs.com/search?q=lodash.isEqual

I would expect that the modularized versions would be the same as the full repo, but I noticed that some of these packages haven't been updated in a while, and I just wanted to make sure that the modularized versions are fully up to date with the full package.

tldr: Using individual lodash package, is lodash.clonedeep as updated as installing lodash and using cloneDeep()?
